pydoctor/epydoc/markup/restructuredtext.py

```diff
@@ -558,9 +386,9 @@ class PythonCodeDirective(Directive):
 
     has_content = True
 
-    def run(self) -> List[Node]:
+    def run(self) -> List[nodes.Node]:
         text = '\n'.join(self.content)
-        node = docutils.nodes.doctest_block(text, text, codeblock=True)
+        node = nodes.doctest_block(text, text, codeblock=True)
         return [ node ]
```

Comment:
It would be good to include these `nodes.` changes in the cleanup PR as well.

Reply:
I assume your talking about the changes made in the following commits:

Use "from docutils import nodes" in node2stan.py
b0968b0

Use "from docutils import nodes" in restructuredtext.py module, too.
a1ec921

Do the same for epytext.py
d4a4df3

Cleanups
3ef32d2

Sorry, I'm not willing to apply this changes on a new clean branch from master, this would be a headache to sort out because this changes almost uniquely applies to lines that needs also changing in this PR.